llvm/test/tools/yaml2obj/GOFF/long-symbol.yaml

```diff
@@ -1,5 +1,7 @@
 # RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
 
+## This tests the long symbol name case requiring continuation records.
```

Comment:
Normally the order in tests is:
```
## Top-level comment
# RUN commands

# CHECK patterns

--- YAML
```
If there are multiple cases within the same test file, things vary a little depending on what typically changes between the test cases.